smarts/env/gymnasium/wrappers/metric/completion.py

```diff
@@ -122,17 +131,15 @@ def _dist_remainder():
     step: int = 0
 
     def func(
-        road_map: RoadMap, obs: Observation, initial_compl: Completion
+        road_map: RoadMap, obs: Dict[str, Any], initial_compl: Completion
     ) -> Completion:
         nonlocal mean, step
 
-        if obs.events.reached_goal:
+        if obs["events"]["reached_goal"]:
             dist = 0
         else:
-            cur_pos = Point(*obs.ego_vehicle_state.position)
-            # pytype: disable=attribute-error
-            goal_pos = obs.ego_vehicle_state.mission.goal.position
-            # pytype: enable=attribute-error
+            cur_pos = Point(*obs["ego_vehicle_state"]["position"])
+            goal_pos = Point(*obs["mission"]["goal_position"])
```

Comment:
We have moved `mission` to `obs["mission"]`. So should we also similarly move `obs.ego_vehicle_state.mission` to `obs.mission` in `Observation` class in `smarts/core/observations.py`?

Reply:
That was not intended, I will revert it to be under "ego_vehicle_state".